src/openproduct/producten/urls.py

```diff
@@ -14,7 +14,21 @@
 description = f"""
 Een API voor Producten.
 
+De Producten Api heeft 1 endpoint `producten`. Hiermee kunnen producten zelf, de bijbehorende eigenaren en documenten worden aangemaakt.
+
+### Product
 Een product is een instantie van een producttype (zie producttypen API), in een product worden onder andere de gegevens van de eigenaar, de benodigde data voor het product en bijvoorbeeld de status vastgelegd.
+Specifieke product data kan worden opgeslagen in de json velden `dataobject` en `verbruiksobject`. Deze velden worden gevalideerd door de `verbruiksobject_schema` & `dataobject_schema` velden van het producttype. ([zie jsonschema](https://json-schema.org))
```

Comment:
Same remark as with `API`
```suggestion
Specifieke product data kan worden opgeslagen in de JSON velden `dataobject` en `verbruiksobject`. Deze velden worden gevalideerd door de `verbruiksobject_schema` & `dataobject_schema` velden van het producttype ([zie jsonschema](https://json-schema.org)).
```